src/Examples/Flowsheets/power_generation/rsofc/cpu.py

```diff
@@ -7,14 +7,26 @@
 # Washington to design NGFC systems with high efficiencies and low CO2
 # emissions.
 ##############################################################################
+###############################################################################
```

Comment:
Minor point, but this probably needs to go first, just so the auto-header too doesn't add it again.